sdl_android/src/main/java/com/smartdevicelink/trace/Mime.java

```diff
@@ -7,6 +7,10 @@
 	private static String m_base64Chars = "ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz0123456789+/";
 	
 	public static String base64Encode(String str) {
+		if(str == null){

```

Comment:
Needs documentation explaining what is returned, and if null is provided, null will be returned.